src/controller/MockController.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import { Project } from "api-builder-types";
+import { FileReaderToJson } from "../Helpers/MethodHelpers";
```

Comment:
Same as https://github.com/Alkesst/Api-Builder-Server-Mock/pull/1/files#diff-ec09168f20a28c6eee296b40b7980dd8

Imported method not used.